interfaces/overview/src/app/components/map/map.component.ts

```diff
@@ -8,6 +9,7 @@ import { Layer, MapService } from 'src/app/services/map.service';
 })
 export class MapComponent implements OnInit {
   public layers: Layer[];
+  public stations: Station[];
```

Comment:
I wonder if there will be only 1 set of 'stations' on the map.
These stations will probably be in "a point layer of (water-)stations".

So the MapService needs to provide a (new) layer with these stations as its contents..
So something like:

```
// this could happen in some `ngOnInit`-step, or triggered by some user-event:
const stations = this.apiService.getStation();
const stationsLayer = this.mapService.createLayer('point', stations);
this.mapService.addLayer(stationsLayer);
```
Or these last steps could be combined differently as:
```
this.mapService.addPointLayerFromArray(stations);

// and the contents of that 👆 would be:
const layer = this.createLayer('point', input);
this.addLayer(layer);
```